README.md

```diff
@@ -146,7 +146,6 @@ To do
 - mktemp
 - mv (almost done, one more option)
 - numfmt
-- od
```

Comment:
Instead of deleting this, can you add "partially finished" or something to that effect?